### ISSUE TITLE
Add prerequisites for other linux distros

### ISSUE THREAD

**Issue by nkprince007** [NIT-dgp/festcms#75]
As of now, we don't have the README tell us about `arch`, `gentoo`, `solus` and other distros.

**spoiler alert** :wink: there are some windows junkies out there too.

**Comment by ghoshbishakh** [comment]
@nkprince007 the requirements.txt should handle all don't you think? Windows also has pip.

**Comment by nkprince007** [comment]
I just meant about README

**Comment by ghoshbishakh** [comment]
@nkprince007 I mean all distros will act as same. Once you have python.

**Comment by nkprince007** [comment]
@ghoshbishakh just about `sqlite3`.

**Comment by ghoshbishakh** [comment]
@nkprince007 we have yum and apt-get. Check the readme

**Comment by nkprince007** [comment]
Yeah, I know. I was talking specifically about pacman, emerge and other package managers. Not fedora and debian
